ceph_bridge.spec

```diff
@@ -0,0 +1,68 @@
+%define pkg_name tendrl-ceph-bridge
+%define pkg_version 0.0.1
+%define pkg_release 1
+
+Name: %{pkg_name}
+Version: %{pkg_version}
+Release: %{pkg_release}%{?dist}
+BuildArch: noarch
+Summary: Tendrl bridge for Ceph Storage
+Source0: %{pkg_name}-%{pkg_version}.tar.gz
+Group:   Applications/System
+License: LGPL2.1
+Url: https://github.com/Tendrl/ceph_bridge
+
+Requires: python-etcd >= 0.4.3
+Requires: python-dateutil >= 2.4.2
+Requires: python-gevent >= 1.0.2
+Requires: python-greenlet >= 0.4.2
+Requires: pytz
+Requires: python-msgpack >= 0.4.5
+Requires: python-taskflow >= 2.6.0
+Requires: tendrl-bridge-common
```

Comment:
Please add `Requires: systemd` as well.

There are no `BuildRequires:` lines here. I think you'll need (at a minimum):

```
BuildRequires: systemd
BuildRequires: python2-devel
BuildRequires: python-sphinx
```

... but you'll need to run this through the "mock" build tool to be certain. Does this package build in mock for EL7? Try rebuilding your SRPM to find out:

```
mock -r epel-7-x86_64 rebuild tendrl-ceph-bridge-0.0.1-1.el7.src.rpm
```
(In fact it would probably be safer if the Makefile used `mock` instead of calling `rpmbuild` directly.)